a6/app/views.py

```diff
@@ -0,0 +1,251 @@
+from django.shortcuts import render
+from django.http import JsonResponse
+from models import *
```

Comment:
it's usually a good practice to import modules that are required.
or do something like,
import models
and then use it like, models.Orders, etc.